modules/portmaster/src/app/pages/monitor/monitor.html

```diff
@@ -1,32 +1,28 @@
 <div tipUpAnchor class="sidebar no-scroll">
   <div class="sidebar-header">
     <div class="row" [class.search-active]="!!(onSearch | async) || showSearch">
-      <h4>
+      <h4 style="display: flex;">
         Network Monitor
-        <app-tipup key="networkMonitor"></app-tipup>
-        <br><span>last 10 minutes</span>
+        <app-tipup style="margin-left: 0.5rem;" key="networkMonitor"></app-tipup>
+        <!-- <br><span>last 10 minutes</span> -->
```

Comment:
Please don't just remove stuff (again!). This needs discussion.

Reply:
Sorry, yea that's my fault. I forgot to uncomment that since I was messing with the icon placement there.